coordinator/coordinator.go

```diff
@@ -132,6 +132,10 @@ func (c *Coordinator) Run() (<-chan struct{}, error) {
 					y: status.ActiveUsers,
 				})
 				latest := getLatestSamples(samples, samplesTimeRange)
+				if len(latest) > 0 {
+					c.log.Debug(fmt.Sprintf("feeback loop info: %d/%d samples, %0.3f slope",
```

Comment:
Should we use structured logging here?

Reply:
I had it structured before and moved to this because it was getting a bit verbose and harder to read.